src/Microsoft.Framework.Runtime/FileGlobbing/PatternsCollectionHelper.cs

```diff
@@ -36,7 +51,7 @@ public static IEnumerable<string> GetPatternsCollection(JObject rawProject, stri
                 return CreateCollection(projectDirectory, token.ValueAsArray<string>());
             }
 
-            throw new InvalidOperationException("Project json doesn't contain qualified token for property " + propertyName + ".");
+            throw new InvalidOperationException("Can't get pattern collection from given type of property: " + token.Type.ToString() + ".");
```

Comment:
Instead of throwing this, use the https://github.com/aspnet/XRE/blob/dev/src/Microsoft.Framework.Runtime/FileFormatException.cs